crates/nu-command/src/database/commands/into_sqlite.rs

```diff
@@ -0,0 +1,334 @@
+                        }
+                    )
+                })
+                .join(",");
+
+            // create the sqlite database table
+            let conn = open_sqlite_db(Path::new(&file.item), file.span)?;
+
+            // create a string for sql table creation
+            let create_statement = format!(
+                "CREATE TABLE IF NOT EXISTS {} ({})",
+                table_name, table_columns_creation
+            );
+
+            // prepare the string as a sqlite statement
+            let mut stmt = conn.prepare(&create_statement).map_err(|e| {
+                ShellError::GenericError(
+                    "Failed to prepare SQLite statement".into(),
+                    e.to_string(),
+                    Some(file.span),
+                    None,
+                    Vec::new(),
+                )
+            })?;
+
+            // execute the statement
+            stmt.execute([]).map_err(|e| {
+                ShellError::GenericError(
+                    "Failed to execute SQLite statement".into(),
+                    e.to_string(),
+                    Some(file.span),
+                    None,
+                    Vec::new(),
+                )
+            })?;
+
+            // use normal sql to create the table
+            // insert into table_name
+            // values
+            // ('xx', 'yy', 'zz'),
+            // ('aa', 'bb', 'cc'),
+            // ('dd', 'ee', 'ff')
+
+            // create the string for inserting data into the table
+            let insert_statement = format!("INSERT INTO {} VALUES {}", table_name, table_values);
+
+            // prepare the string as a sqlite statement
+            let mut stmt = conn.prepare(&insert_statement).map_err(|e| {
+                ShellError::GenericError(
+                    "Failed to prepare SQLite statement".into(),
+                    e.to_string(),
+                    Some(file.span),
+                    None,
+                    Vec::new(),
+                )
+            })?;
+
+            // execute the statement
+            stmt.execute([]).map_err(|e| {
+                ShellError::GenericError(
+                    "Failed to execute SQLite statement".into(),
+                    e.to_string(),
+                    Some(file.span),
+                    None,
+                    Vec::new(),
+                )
+            })?;
+
+            // and we're done
+            Ok(Value::Nothing { span: *span })
+        }
+        _ => Err(ShellError::UnsupportedInput(
+            format!(
+                "Expected a list but instead received a {}",
+                input.get_type()
+            ),
+            span,
+        )),
+    }
+}
+
+// This is taken from to text local_into_string but tweaks it a bit so that certain formatting does not happen
+fn nu_value_to_string(value: Value, separator: &str, config: &Config) -> String {
```

Comment:
There is a into string function for value. No?

Reply:
There are a bunch of these type of functions but I couldn't find one that did exactly what I needed to do.